packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -114,16 +119,16 @@ class ConfigManager implements IConfigManager {
     }
 
     let res: SourceConfigResponse;
-    const errMessage = 'Unable to process/parse source config';
+    const errMessage = 'Unable to process/parse source configuration response';
```

Comment:
should we move it in the log messages constants file?

Reply:
Good call. Done.